elasticdl/python/master/k8s_instance_manager.py

```diff
@@ -101,7 +101,7 @@ def __init__(
 
         # Protects followed variables, which are accessed from event_cb.
         self._lock = threading.Lock()
-        # worker id to (pod name, phase) mapping
+        # worker id to (pod name, ip, phase) mapping
```

Comment:
We can get pod by pod name and get IP by `pod.status.IP`. Should we need to set IP here?